modules/core_project_factory/main.tf

```diff
@@ -26,18 +26,17 @@ resource "random_id" "random_project_id_suffix" {
  *****************************************/
 locals {
   group_id               = "${var.manage_group ? format("group:%s", var.group_email) : ""}"
-  project_id             = "${google_project.main.project_id}"
-  project_number         = "${google_project.main.number}"
```

Comment:
Why drop `local.project_number`?

Reply:
The local here is a level of indirection that is unnecessary, IMHO.

Reply:
It's easier to manage this in one place than every place.